website/board_games/views.py

```diff
@@ -28,7 +30,17 @@ def advanced_search(request):
                 games = games.filter(game_length_max__lte=form.cleaned_data['game_length_max'])
             if form.cleaned_data['tag']:
                 games = games.filter(tag__icontains=form.cleaned_data['tag'])
-            return render(request, 'board_games/advanced_search.html', context={'form': form, 'simple_search': simple_search, 'games': games})
+            games_and_tags = {}
+            for tag in tags:
+                if tag.game_id in games:
+                    if tag.game_id not in games_and_tags:
+                        games_and_tags[tag.game_id] = ""
+                        print(tag.game_id)
+                    games_and_tags[tag.game_id] += tag.tag_id + ", " 
+            for key in games_and_tags.keys():
+                games_and_tags[key] = games_and_tags[key][:-2]
+            print(games_and_tags)
```

Comment:
```suggestion
                    games_and_tags[tag.game_id] += tag.tag_id + ", " 
            for key in games_and_tags.keys():
                games_and_tags[key] = games_and_tags[key][:-2]
```
Faut penser à enlever les prints de debug